openbas-api/src/main/java/io/openbas/rest/inject/InjectApi.java

```diff
@@ -170,6 +172,13 @@ public Inject tryInject(@PathVariable String injectId) {
     return atomicTestingService.tryInject(injectId);
   }
 
+  @GetMapping(INJECT_URI + "/{injectId}/executable-payload")
```

Comment:
Could you add a Tracing annotation (you can have a look at what's already in ExerciseApi.java for an example) ?

Reply:
Also, if you want to test, if your dev environment's docker-compose is up to date, you can reach localhost:16686. I'll do a demo whenever it's available for prod envs.